src/msw/statbox.cpp

```diff
@@ -39,6 +39,9 @@
 #include "wx/sysopt.h"
 
 #include "wx/msw/uxtheme.h"
+
+#include <windowsx.h> // needed by GET_X_LPARAM and GET_Y_LPARAM macros
```

Comment:
We need to `#undef Delete{Pen,Brush,...}` which are defined as `DeleteObject()` in `windowsx.h` now...